language/move-lang/tests/move_check/parser/hexstring_bad_value2.exp

```diff
@@ -0,0 +1,8 @@
+error: 
+
+   ┌── tests/move_check/parser/hexstring_bad_value2.move:3:9 ───
+   │
+ 3 │         x"abcdefg"
+   │         ^^^^^^^^ Missing closing quote (") after hex string
```

Comment:
This error message looks wrong. Shouldn't it be complaining about the `g`?

Reply:
Ditto, I feel like we should take the string, and then try to parse it as a hex string. Then give the error.

Reply:
Yeah, I know -- I wasn't very happy with this message, either. I hadn't yet come up with a better solution, but I like Todd's idea.